### ISSUE TITLE
install-pwr-service.sh powers Pi off, OS won't boot

### ISSUE THREAD

**Issue by vlig** [geekworm-com/x735-v2.5#20]
```
$ sudo ./install-pwr-service.sh
Start installing the power management service...
Power management service installed
$
Broadcast message from root@pi41 (Sat 2024-09-28 12:45:57 MSK):

The system will power off now!
```
Doing steps according the manual. Then the OS isn't booting, just shows the logo, then powers off again. Only deleting the /lib/systemd/system/x735-pwr.service file from the microsd card makes the OS bootable again.

P.S. Please, translate Chinese to English in the repo files.

**Comment by geekworm-com** [comment]
This project is deprecated, please refer to the new repository https://github.com/geekworm-com/x735-script

**Comment by vlig** [comment]
> This project is deprecated, please refer to the new repository https://github.com/geekworm-com/x735-script

Yes, I'm using it certainly, sorry. I've just written to the wrong repo. I'll make my addition there, thanks.
